src/apis/abusiveexperiencereport/v1.ts

```diff
@@ -167,6 +167,55 @@ export namespace abusiveexperiencereport_v1 {
     /**
      * abusiveexperiencereport.sites.get
      * @desc Gets a site's Abusive Experience Report summary.
+     * @example
+     * // Before running the sample:
+     * // - Enable the API at:
+     * //   https://console.developers.google.com/apis/api/abusiveexperiencereport
+     * // - Login into gcloud by running:
+     * //   `$ gcloud auth application-default login`
+     * // - Install the npm module by running:
+     * //   `$ npm install googleapis`
+     *
+     * const {google} = require('googleapis');
+     * const abusiveexperiencereport = google.abusiveexperiencereport('v1');
+     *
+     * async function main() {
+     *   const auth = new google.auth.GoogleAuth({
+     *     // Scopes can be specified either as an array or as a single, space-delimited string.
+     *     scopes: [],
+     *   });
+     *
+     *   // Acquire an auth client, and bind it to all future calls
+     *   const authClient = await auth.getClient();
+     *   google.options('auth', authClient);
+     *
+     *   // Do the magic
+     *   const res = await abusiveexperiencereport.sites.get({
+     *     // Required. The name of the site whose summary to get, e.g.
+     *     // `sites/http%3A%2F%2Fwww.google.com%2F`.
```

Comment:
Is this supposed to be like this, or do we need `| safe`?  (I really don't know)

Reply:
So I *do* use `| safe`.  Turns out this is exactly what's in the source discovery doc 🤦